js/modules/k6/ws/ws_test.go

```diff
@@ -236,6 +236,24 @@ func TestSessionBadInterval(t *testing.T) {
 	require.Contains(t, err.Error(), "setInterval requires a >0 timeout parameter, received -1.23 ")
 }
 
+func TestSessionIntervalSub1(t *testing.T) {
```

Comment:
```suggestion
func TestSocketSetInterval(t *testing.T) {
```
If the current name is meaningful then I'm not getting it.

Reply:
I kind of prefer to have it somehow mention that it is testing a very small interval as that was what was breaking. But am okay with a different name - I did not like this one either way, I just needed something to have a test to confirm what I was suspecting